docs/extensions/module_rating.rst

```diff
@@ -0,0 +1,83 @@
+
+Tripal Module Rating System
+=============================
+
+This module rating system is meant to aid Tripal Site Administrators in choosing extension modules for their site. It is also meant to guide developers in module best practices and celebrate modules which achieve these goals.
+
+Bronze
+-------
+
+.. image:: Tripal-Bronze.png
+
+- Has a public release.
+- Should install on a Tripal site appropriate for the versions it supports.
+- Defines any custom tables or materialized views in the install file (if applicable).
+- Adds any needed controlled vocabulary terms in the install file (Tripal3)
+- Provides Installation and admin instructions README.md (or `RTD <https://tripal.readthedocs.io/en/latest/dev_guide/rtd.html>`_)
+- Has a license (distributed with module)
+
+Silver
+-------
+
+.. image:: Tripal-Silver.png
+
+- Follows basic Drupal Coding standards; specifically, `code format <https://www.drupal.org/docs/develop/standards/coding-standards>`_ and `API documentation <https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#drupal>`_
+- Uses Tripal API functions. Specifically, it should use the
+    - `Chado Query API <http://api.tripal.info/api/tripal/tripal_chado%21api%21tripal_chado.query.api.inc/group/tripal_chado_query_api/3.x>`_ for querying chado.
```

Comment:
Given that tripal now supports different storage systems, I think we should add `if applicable` here somewhere.

Reply:
Good point!